src/routes/feed.xml.ts

```diff
@@ -0,0 +1,149 @@
+import mm from 'music-metadata';
+
+interface Episode {
+	title: string;
+	description: string;
+	fileLocation: string;
+	length: number;
+	duration: number;
+	season: number;
+	number: number;
+	published: Date;
+	explicit: boolean;
+	episodeImageURL?: string;
+	guid: string;
+}
+
+interface Channel {
+	title: string;
+	url: string;
+	author: string;
+	email: string;
+	albumImageURL: string;
+	description: string;
+	keywords: string[];
+	category: string;
+	explicit: boolean;
+}
+
+const manifest = [
+	'001-WelcomeToTheForum.mp3',
+	'002-WhatsInAName.mp3',
+	'003-QuietMajorities.mp3',
+	'004-DontCopyThatFloppy.mp3',
+	'005-DontReadTheComments.mp3',
+	'006-BalancingTheScale.mp3',
+	'007-OwnYourView.mp3',
+];
+
+//TODO apple itunes allows for multi/subcategories but I'm being lazy for us
+//Apple spec from here: https://help.apple.com/itc/podcasts_connect/#/itcb54353390
+//TODO implement all optional/situational tags as well
+const xml = (channel: Channel, episodes: Episode[]) => `
+<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd" xmlns:googleplay="http://www.google.com/schemas/play-podcasts/1.0" xmlns:media="http://www.rssboard.org/media-rss" version="2.0">
+<atom:link href="${channel.url}/feed.xml" rel="self" type="application/rss+xml"/>
+<channel>
+	<title>${channel.title}</title>
+	<itunes:title>${channel.title}</itunes:title>
+	<link>${channel.url}</link>
+	<language>en-us</language>
+	<copyright>&#169; 2022 ${channel.author}</copyright>
+	<itunes:author>${channel.author}</itunes:author>
+	<description>${channel.description}</description>
+	<itunes:type>episodic</itunes:type>
+	<itunes:owner>
+		<itunes:name>${channel.author}</itunes:name>
+		<itunes:email>${channel.email}</itunes:email>
+	</itunes:owner>
+	<itunes:image href="${channel.albumImageURL}"/>
+	<itunes:category text="${channel.category}"></itunes:category>
+	<itunes:explicit>${channel.explicit ? 'YES' : 'NO'}</itunes:explicit>
+
+${episodes
+	.map(
+		(episode) => `
+	<item>
+		<itunes:episode>${episode.number}</itunes:episode>
+		<itunes:episodeType>full</itunes:episodeType>
+		<itunes:season>${episode.season}</itunes:season>
+		<title>${episode.title}</title>		
+		<itunes:title>${episode.title}</itunes:title>
+		<description>
+			<![CDATA[
+			${episode.description}
+			]]>
+		</description>
+		<enclosure 
+			url="${episode.fileLocation}" 
+			length="${episode.length}" 
+			type="audio/mpeg"/>
+		<guid isPermaLink="false">${episode.guid}</guid>
+		<pubDate>${episode.published}</pubDate>
```

Comment:
```suggestion
		<pubDate>${episode.published.toUTCString()}</pubDate>
```

Looks like the feed will change based on the machine timezone that's generating it